src/library/FauxControl/FauxControl.js

```diff
@@ -59,6 +59,8 @@ const getIcons = ({
 };
 
 export default function FauxControl({
+  borderColor,
+
```

Comment:
remove empty line

Reply:
@tx-michael  I think the blank line is to more easily distinguish our customizations from what was in the original package without having to look at the diff.